src/lib/bootloader/stage1_device.rb

```diff
@@ -57,24 +56,29 @@ def underlaying_devices_for(dev)
       res.uniq
     end
 
+    def pv_disk(vgs)
+      pvs = usable_pvs(vgs)
+      pvs.map { |p| p.ancestors.find { |a| a.is?(:disk) }.name }
+    end
+
     # get one level of underlaying devices, so no recursion deeper
     def underlaying_devices_one_level(dev)
       # FIXME: there is nothing like this right now in libstorage-ng
       #  a_vg.name #=> "/dev/vgname"
```

Comment:
quite sad, I hope when device exist, it should have name. It will cause troubles for us in more complex scenarios.